modules/ROOT/pages/monitoring-dashboards.adoc

```diff
@@ -1,78 +1,66 @@
-= Monitoring Dashboards
+= Monitoring Dashboards on Runtime Manager
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
-:keywords: cloudhub, analytics, monitoring, insight, filter
 
-image:logo-cloud-active.png[link="/runtime-manager/deployment-strategies", title="CloudHub"]
-image:logo-hybrid-active.png[link="/runtime-manager/deployment-strategies", title="Hybrid Deployment"]
-image:logo-server-disabled.png[link="/runtime-manager/deployment-strategies", title="Anypoint Platform Private Cloud Edition"]
-image:logo-pcf-disabled.png[link="/runtime-manager/deployment-strategies", title="Pivotal Cloud Foundry"]
 
+The builtin monitoring dashboard gives you in-depth visibility into usage metrics about both your deployed apps and the systems they're deployed to. The dashboard gives you a view into the evolution of metrics about your apps. 
```

Comment:
built-in; but I'd delete "built-in" because it makes "in-depth" distracting repetition of form. Just me?

Reply:
Should "dashboard" be plural since you mention two different dashboards below? We may know it's all the same software just showing different things, but to customer, since we name it, it's a thing different from the thing with a different name...?